### ISSUE TITLE
Feature: User configurable shortcuts

### ISSUE THREAD

**Issue by nomorsug** [usebruno/bruno#561]
[JIRA](https://usebruno.atlassian.net/browse/BRU-2414)

A new tab in Preferences where you can change/reset shortcuts like `Keyboard Shortcuts` in a table:

|      Command               | Default Keybinding | User Keybinding | Reset
| ------------------- | -------------------- | ------------------ | --|
| **Close Tab**     | Ctrl+W                 |             | reset button

...

It can also act as documentation. The user configuration can be stored in a JSON file that can be imported/exported:

```javascript
{
  {
       "command": "closeTab",
       "key": "ctrl+w",             
       "key_user": "" 
  }
}
```

**Comment by bukajsytlos** [comment]
I would love to migrate to Bruno, but not having shortcuts to switch recent requests and search for requests is block as I have pretty large collection.
Thx

**Comment by skreutzberger** [comment]
Yes that feature would be super important! For example I am using keyboard shortcuts in all my apps to move between tabs. Unfortunately my Macbook keyboard does not have PageUp and PageDown so I can not set the currently fix shortcuts. A simple env file would be enough for me

**Comment by gssj85** [comment]
I would love being able to change the current keybindings too, the Send Request actually requires to leave the hand from the mouse to press ENTER key, I would prefer to use a one-hand shortcut like CTRL+D or CTRL+SPACE.

**Comment by gabriellanata** [comment]
Adding my +1 here. In ost of my apps I have cmd+R set up to "Run" and I would love to be able to change that in Bruno for consistency

**Comment by arthurvanl** [comment]
+1

**Comment by ClaireMaud** [comment]
+1

**Comment by semihcosu** [comment]
+1 Would make it much easier to migrate from other apps

**Comment by A-Aidyn** [comment]
+1

**Comment by Yaquelqun** [comment]
Adding a +1, i'd like to be able to disable shortcuts because i use ctrl + = (on a mac) to show an overlay terminal and if i do it with bruno in the background it just keeps getting more and more zoomed in

**Comment by anusree-bruno** [comment]
Hey we are supporting customization in v3.3.0. Please do check it out https://www.usebruno.com/downloads
Changelog: https://www.usebruno.com/changelog
